otel/propagator_test.go

```diff
@@ -28,37 +28,31 @@ type transactionTestContext struct {
 	sampled sentry.Sampled
 }
 
-type otelTestContext struct {
-	traceId    string
-	spanId     string
-	traceFlags trace.TraceFlags
-}
-
-func createTransactionAndMaybeSpan(transactionContext transactionTestContext, withSpan bool) {
-	client, _ := sentry.NewClient(sentry.ClientOptions{
-		Dsn:           "https://abc@example.com/123",
-		Environment:   "testing",
-		Release:       "1.2.3",
-		EnableTracing: true,
-	})
-	hub := sentry.NewHub(client, sentry.NewScope())
-	ctx := sentry.SetHubOnContext(context.Background(), hub)
-
+func createTransactionAndMaybeSpan(transactionContext transactionTestContext, withSpan bool) trace.SpanContextConfig {
 	transaction := sentry.StartTransaction(
-		ctx,
+		emptyContextWithSentry(),
```

Comment:
```suggestion
		emptyCtxWithSentry(),
```

Reply:
It's a helper function that's used in multiple places already, do we really want this?

Reply:
Up to you, no strong feelings.